docs/pages/reference/access-controls/roles.mdx

```diff
@@ -371,10 +371,9 @@ an administrative role. In particular, you should avoid configuring `allow` rule
 that grant `create` and `update` permissions on `token` resources to prevent
 unexpected changes to the configuration or state of your cluster.
 
-## RBAC for sessions
+## RBAC for session recordings
```

Comment:
Should we also link to this on the "further reading" section of session-recording.mdx?